libs/ipfs-http-client/Makefile

```diff
@@ -0,0 +1,73 @@
+# 
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipfs-http-client
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Leonid Esman <leonid.esman@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/vasild/cpp-ipfs-http-client.git
+PKG_SOURCE_DATE:=2019-11-05
+PKG_SOURCE_VERSION:=763e59ad698f3e3846f85df11e01c18ef3fbc401
+PKG_MIRROR_HASH:=a2b5721faf0d43ddb4a892245ef382666149c83f3f97e558e1a6acf2402fb9fd
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ipfs-http-client/Default/description
+  IPFS (the InterPlanetary File System) is the Distributed Web. 
+  Specs, docs, sources, links: https://ipfs.io/ and https://github.com/ipfs.
+  This is Vasil Dimov's C++ IPFS HTTP API client library.
+endef
+
+define Package/libipfs-http-client
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=IPFS client library
+  URL:=https://github.com/vasild/cpp-ipfs-http-client
+  DEPENDS:= +libcurl +libstdcpp
+  PKG_BUILD_DEPENDS:=nlohmannjson
```

Comment:
This does not belong here

Reply:
Why?  I don't understand what you want to say.

Reply:
It belongs outside of the group.

Reply:
Line PKG_BUILD_DEPENDS moved outside of the group